dotcom-rendering/src/components/HeaderTopBar.importable.tsx

```diff
@@ -69,10 +69,8 @@ export const HeaderTopBar = ({
 	headerTopBarSearchCapiSwitch,
 	isInEuropeTest,
 }: HeaderTopBarProps) => {
-	const isServer = typeof window === 'undefined';
-	const isSignedIn =
-		!isServer && !!getCookie({ name: 'GU_U', shouldMemoize: true });
-
+	const isUserSignedIn = useIsSignedIn();
+	console.log('top bar isSignedIn', isUserSignedIn);
```

Comment:
I'm not sure if we want to use the logging library from https://github.com/guardian/csnx/tree/main/libs/%40guardian/libs/src/logger?